go/saltpack/specs/saltpack_encryption.md

```diff
@@ -160,72 +187,79 @@ recipients, then she will have the key, and she could modify the payload. The
 **authenticators list** is there to prevent this attack, protecting the
 recipients from each other.
 
-We compute each recipient's authenticator in three steps:
+We compute the authenticators in three steps:
 
-- Compute the SHA512 of the **payload secretbox**.
-- Encrypt that hash in a NaCl box, using the sender and recipient's long-term
-  keys. See [Nonces](#nonces) below.
-- The authenticator is the first 16 bytes of that box.
+- Serialize the packet number as a 64-bit unsigned big-endian integer, where
```

Comment:
Just use the payload box nonce here, so 16 zeros before the 64-bit unsigned big-endian integer.